packages/patterns-react/src/patterns/CardArray/__stories__/CardArray.stories.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { DDS_CARDARRAY } from '../../../internal/FeatureFlags';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+import '../../../../../styles/scss/patterns/cardarray/index.scss';
```

Comment:
@kennylam correct me if I'm wrong, we want to move this so that we include a story level `scss` file, which then imports the `cardarray` styles (and any other necessary styles), since webpack has an issue loading these in on the JS side and not tracking if they have already been imported?

Reply:
@jeffchew yep that's right.

@RaphaelAmadeu you want to replace those imports with just one `index.scss`, and in that file you would have your `cardarray` import.

in your story:
`import './index.scss';`

in `index.scss`
`@import '../../../../../styles/scss/patterns/cardarray/index.scss';`

Reply:
@RaphaelAmadeu I think I might have missed this in the Pictogram one, can you check and create a separate PR for that pattern?

Reply:
Sure thing, @kennylam thanks for the clarification.